### PR TITLE
Created Candidate Dashboard + Organization Dashboard 

### PR DESCRIPTION
**Pull request by Megha-Pathak**
Hello @devesh-verma, 

This PR caries the initial code for the candidate dashboard and organization dashboard. 
Let us know your feedback on this. 

The authentication code is also almost ready, will commit it soon.

### CODE REVIEW

File: client/src/Components/Dashboard/DashboardContent.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        >
+          Siddharth S
+        </Typography>
+        <Typography
+          className={classes.pos2}
+          variant="h7"
+          component="p"
+          align="center"
+        >
+          siddharth.simharaju@gmail.com
```
Comment (vaibhavdaren) [inline client/src/Components/Dashboard/DashboardContent.jsx:59]: is this static?

Comment (Megha-Pathak) [reply client/src/Components/Dashboard/DashboardContent.jsx:59]: Yes

**Comment by Megha-Pathak** [comment]
> 
> 
> I think you can pull hardcoded data out into variables in a separate file data.json and load it as if you were calling an API. when the backend is ready you can pull these off really easily.

can I have the API structure (an example response from API) for pulling our the data in a constant or I shall create it myself?

**Comment by vaibhavdaren** [comment]
> can I have the API structure (an example response from API) for pulling our the data in a constant or I shall create it myself?

resolved

**Pull request merged**